test/e2e/util.go

```diff
@@ -463,6 +463,21 @@ func addressIsIP(address v1.NodeAddress) bool {
 	return true
 }
 
+func formatAddressAndPort(addr string, port uint16) (string, error) {
```

Comment:
@ricky-rav this is basically `net.JoinHostPort` right?

Reply:
Spot on! Let me replace it, then.